src/audio.cpp

```diff
@@ -0,0 +1,53 @@
+
+/*
+  In this example, the Arduino simulates the sampling of a sinusoidal 1000 Hz
+  signal with an amplitude of 100, sampled at 5000 Hz. Samples are stored
+  inside the vReal array. The samples are windowed according to Hamming
+  function. The FFT is computed using the windowed samples. Then the magnitudes
+  of each of the frequencies that compose the signal are calculated. Finally,
+  the frequency with the highest peak is obtained, being that the main frequency
+  present in the signal.
+*/
+#include "audio.h"
+
+void microphone_iteration()
+{
+
+  /*
+  These values can be changed in order to evaluate the functions
```

Comment:
?

Reply:
this is a critical part of the implemenation and is crutuial for the fungtion of the